docs/user-manual/graphics/shaders/glsl-specifics.md

```diff
@@ -0,0 +1,56 @@
+---
+title: GLSL Specifics
+sidebar_position: 0
+---
+
+GLSL shaders used by the PlayCanvas engine must satisfy certain requirements. These requirements allow the engine to correctly integrate shaders, ensuring they receive the necessary resources such as attributes, uniforms, and varyings. Following these rules also allows us to automatically process the shader for slightly different requirement for their transpilation to WGSL used by the WebGPU.
```

Comment:
```suggestion
GLSL shaders used by the PlayCanvas engine must satisfy certain requirements. These requirements allow the engine to correctly integrate shaders, ensuring they receive the necessary resources such as attributes, uniforms, and varyings. Following these rules also allows us to automatically process the shader for slightly different requirement for their transpilation to WGSL used by WebGPU.
```

Reply:
This sentence doesn't read right: "Following these rules also allows us to automatically process the shader for slightly different requirement for their transpilation to WGSL used by WebGPU."